ui/src/app/cohort-review/detail-tab-table/detail-tab-table.component.tsx

```diff
@@ -328,18 +354,17 @@ export const DetailTabTable = withCurrentWorkspace()(
         counts.total++;
       });
       let options: Array<any>;
-      switch (colName) {
-        case 'standardVocabulary':
-          /* TODO need to check for Source also after adding the standard/source radio buttons */
-          const vocabs = vocabOptions.getValue().Standard;
-          options = vocabs[domain] ? vocabs[domain].map(option => {
-            return {name: option, count: counts[option] || 0};
-          }) : [];
-          break;
-        case 'domain':
-          options = domains.map(option => {
-            return {name: option, count: counts[option] || 0};
-          });
+      if (colName === 'domain') {
+        options = domains.map(option => {
+          return {name: option, count: counts[option] || 0};
+        });
+      } else {
+        const vocabs = colName === 'standardVocabulary'
+          ? vocabOptions.getValue().Standard
+          : vocabOptions.getValue().Source;
+        options = vocabs[domain] ? vocabs[domain].map(option => {
+          return {name: option, count: counts[option] || 0};
+        }) : [];
       }
       options.push({name: 'Select All', count: counts.total});
       if (checkedItems[colName].find(i => i === 'Select All')) {
```

Comment:
On line number 376 where we have overlay for filter we should check data!==null  then only show.